src/components/Annotation/Popup/Popup.tsx

```diff
@@ -39,10 +39,8 @@ export const Popup = (props: PopupProps) => {
   const hasBodies = selected.bodies.length > 0;
 
   // Close the popup after a reply
-  const onReply = (body: AnnotationBody) => {
-    store.addBody(body);
```

Comment:
Annotation body got added twice. A regression artefact from my refactoring, which moved store CRUD ops up the React component hierarchy.